### DISCUSSION TITLE
Welcome to LLTHW Discussions!

### DISCUSSION THREAD

**Discussion by thephoeron** [thephoeron/LLTHW#45] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by phoe** [comment]
Oh snap, GitHub has some sorta forum now? I wasn't aware of that.

Hey, I'm phoe, I'm a Lisp hacker. I guess I'm not a noob anymore, which means that I can actually help build LLTHW in some way.

**Reply by thephoeron** [reply]
Wild right?
